### PR TITLE
build_cpp.sh: remove duplicate -DCMAKE_BUILD_TYPE and be more verbose while compiling

### PR DESCRIPTION
**Pull request by meyerj**

### CODE REVIEW

File: build_cpp.sh [hunk 1]
```diff
@@ -108,7 +107,7 @@ catkin config \
     -DCMAKE_FIND_ROOT_PATH=$prefix -DCMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH \
     -DBUILD_TESTING=OFF -DCATKIN_ENABLE_TESTING=OFF
 
-catkin build
+catkin build --force-cmake --summary -ip1
```
Comment (jubeira) [inline build_cpp.sh]: This script has an unused `VERBOSE` flag above. Can we use it to set `-ip1`?

**Review by jubeira — APPROVED** [review]
Just a minor comment, otherwise LGTM!

File: build_cpp.sh (hunk 1 above)
Comment (meyerj) [reply build_cpp.sh]: Done: https://github.com/Intermodalics/ros_android/pull/69/commits/d07817b30ed342443a203319fce634e084c38e02

**Comment by meyerj** [comment]
I added some more extensions useful for debugging, e.g. to pass custom catkin arguments to `build_cpp.sh`.

- `CMAKE_PREFIX_PATH` and `CMAKE_INSTALL_PREFIX` cmake variables do not need to be set explicitly as they will be added by catkin_tools depending on the workspace configuration (`catkin config`).
- `USE_CATKIN` was also unused - at least `grep USE_CATKIN catkin_ws/src -R` does not return any result.
- Removed `ENTRYPOINT /ros_entrypoint.sh` defined in the docker base image such that the environment is clean and the build does not accidentally pick up the CMAKE_PREFIX_PATH set in the shell, e.g. when calling `build_cpp.sh` directly.

**Comment by meyerj** [comment]
@jubeira Still good to merge? Some significant changes have been added since your [last review](https://github.com/Intermodalics/ros_android/pull/69#pullrequestreview-184158295).

**Comment by jubeira** [comment]
Yes, I reviewed them yesterday and left it building from scratch, but I had to leave before it finished. I had no issues and the new changes are looking good; merging now!

**Pull request merged**